gtfs/__main__.py

```diff
@@ -1,25 +1,42 @@
 #!/usr/bin/env python
 """Command line interface for fetching GTFS."""
-import argparse
-import getpass
+import typer
+from chalky import chain
 import logging
-import sys
-
 from prettytable import PrettyTable
-
-from gtfs.utils.FeedSource import FeedSource
+from halo import Halo
+import importlib
+from utils.FeedSource import FeedSource
 import feed_sources
-# import all the available feed sources
-# pylint: disable=I0011,wildcard-import
-from feed_sources import *
 
 logging.basicConfig()
 LOG = logging.getLogger()
 LOG.setLevel(logging.INFO)
+app = typer.Typer()
+sources = list(feed_sources.__all__)
```

Comment:
what does this return? the file names? probably best to import each FeedSource into `feed_sources.__init__.py` and define a `__all__` in that file with a list of all classes imported there. i.e.

feed_sources/__init__.py
```
from .Berlin import Berlin

__all__ = ["Berlin"]
```
or so?

Then you don't have to import it further down anymore

Reply:
Yup that sounds good. I was unfamiliar with the use of init.py that's why I didn't think of it before.